site/docs/v1/tech/samlv2/tableau-cloud.adoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: doc
+title: SAML v2 & Tableau Cloud
+description: Setting up Tableau Cloud to login using FusionAuth via SAML v2
+navcategory: login-methods
+---
+
+== Overview
+
+link:https://www.tableau.com/[Tableau] is one of the most advanced data visualization tools used for business intelligence and data analysis. This document covers the configuration necessary to get Tableau Cloud working with FusionAuth as the identity provider using SAML v2, allowing users to log in to your Tableau Cloud account using their FusionAuth credentials.
+
+=== Prerequisites
+
+This document assumes you have a running instance of FusionAuth and a working Tableau Cloud application. You will also need admin accounts for both to configure them correctly.
+
+Finally, you'll need a FusionAuth user that you will use to sign into Zendesk. You can use an existing user or create a new user for this purpose.
+
+=== Instructions
+
+We also recommend taking a look at link:https://help.tableau.com/current/online/en-us/saml_config_site.htm[Tableau Cloud SSO docs], but here are the instructions.
```

Comment:
> Can we submit this to them once it is published?

I've [posted a question in their forums](https://community.tableau.com/s/idea/0878b000000A2JHAA0/detail) and they said we could have it as a [KB article](https://www.tableau.com/support/knowledgebase) instead of the official docs.

> Why do we recommend them looking at the instructions?

I have this habit of having something like "please take a look at their docs" just in case they ever change their UI and our guide becomes outdated, but maybe I should put it in a Reference section or something. What do you think?